compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeBuilder.java

```diff
@@ -5430,15 +5430,23 @@ private BLangIdentifier createIdentifier(Token token) {
         return createIdentifier(getPosition(token), token);
     }
 
+    private BLangIdentifier createIdentifier(Token token, boolean isXML) {
```

Comment:
We don't need this method right? We can directly call `createIdentifier(Location pos, Token token, boolean isXML)` from `transform(XMLSimpleNameNode xmlSimpleNameNode)`

Reply:
Corrected 234ff29a48d70dff3c6c5992ff72a7873ed7113a